catalog/app/containers/Bucket/PackageCopyDialog.js

```diff
@@ -189,17 +241,27 @@ function DialogForm({
                 margin="normal"
               />
 
-              <RF.Field
-                component={FilesInput}
-                name="files"
-                validate={validators.nonEmpty}
-                validateFields={['files']}
-                errors={{
-                  nonEmpty: 'Add files to create a package',
-                }}
-                isEqual={R.equals}
-                initialValue={initialFiles}
-              />
+              <div className={classes.filesWrapper}>
+                <RF.Field
+                  className={classes.filesSwitcher}
+                  component={CopyDataSwitcher}
+                  name="copyData"
+                  initialValue
+                  validateFields={['copyData']}
+                />
+
+                <RF.Field
+                  component={FilesInput}
+                  name="files"
+                  validate={validators.nonEmpty}
```

Comment:
all this logic here (validation, error handling, equality checks, handling the actual field state) is unused, bc files are not editable